src/libraries/System.Net.Sockets/tests/FunctionalTests/CreateSocketTests.cs

```diff
@@ -409,7 +410,6 @@ public async Task Ctor_SafeHandle_Tcp_SendReceive_Success(AddressFamily addressF
             Assert.Equal(42, buffer[0]);
         }
 
-        [PlatformSpecific(TestPlatforms.Windows | TestPlatforms.Linux)] // OSX/FreeBSD doesn't support SO_ACCEPTCONN, so we can't query for whether a socket is listening
```

Comment:
Isn't `proc_pidfdinfo()` something that's macOS-specific? Shouldn't this also be skipped on FreeBSD? (Or SO_ACCEPTCON now works there too?)

Reply:
I will deal with FreeBSD later. I have PR up for builds. I plan to run all tests and either fix failures or open issues for tracking. (or close existing one if we get more passes)